packages/data-context/src/codegen/spec-options.ts

```diff
@@ -3,11 +3,14 @@ import type { CodeGenType } from '@packages/graphql/src/gen/nxs.gen'
 import type { WizardFrontendFramework } from '@packages/scaffold-config'
 import fs from 'fs-extra'
 import path from 'path'
+import type { FoundSpec, TestingType } from '@packages/types'
 
 interface CodeGenOptions {
   codeGenPath: string
   codeGenType: CodeGenType
   isDefaultSpecPattern: boolean
+  specPattern: string[]
+  currentProject: string | null
   erroredCodegenCandidate?: string | null
```

Comment:
Now that we support custom spec pattern, can we confirm that there is no use for `erroredCodegenCandidate`? I'd really like to see this removed as it's pretty confusing.

Reply:
Yeah honestly I've held off on removing the logic surrounding that because I'm not sure what it does. I'll take a look at it now and see about removing it